miner/worker.go

```diff
@@ -57,6 +58,8 @@ var (
 
 	txConditionalRejectedCounter = metrics.NewRegisteredCounter("miner/transactionConditional/rejected", nil)
 	txConditionalMinedTimer      = metrics.NewRegisteredTimer("miner/transactionConditional/elapsedtime", nil)
+
+	txInteropRejectedCounter = metrics.NewRegisteredCounter("miner/transactionInterop/rejected", nil)
```

Comment:
Do we want to track other metrics here? Like successful messages, or RPC interaction successes/failures?

Reply:
This was from the base PR which was merged already. There's a monorepo issue open to track polish with additional metrics